src/components/CardInfo.tsx

```diff
@@ -23,6 +23,8 @@ type CardInfoApiData = {
   id: number;
 };
 
+const posterPlaceholder = '/images/movie-placeholder.jpg';
```

Comment:
instead of changing this 3 places you better set check here https://github.com/oleksandrstarov/sigma_js_internship_2021/blob/6cf45ed07d0857b0557d0d394fe24c98a9343129/src/service/api.tsx#L188

`
getFullImgLink(url: string, size: string = 'w500') {
    if(!url) {
        posterPlaceholder
    }
    return `${API_IMG_URL}${size}${url}`;
  },
`

Reply:
Yep, changed it

Reply:
![image](https://user-images.githubusercontent.com/56648514/116143406-c7837400-a6e3-11eb-927d-2d2412d375e1.png)

I cant understand how to avoid this problem